zh-CN/mapping_dev.zh_CN

```diff
@@ -2195,18 +2195,18 @@ wndLog.tbtInvestigate=使用 DeepGit 逐行调查历史记录。
 wndLog.tbtMerge=合并来自其他分支的更改。
 wndLog.tbtPush=将本地提交推送到远程源仓库。
 wndLog.tbtRebaseHeadTo=将 HEAD 提交应用于所选提交。
-wndLog.tbtRevealWorkingTree=Reveal working tree node in graph.
+wndLog.tbtRevealWorkingTree=在图中显示工作树节点。
 wndLog.tbtStage=将工作树文件暂存在索引中以准备下一次提交。
 wndLog.tbtStashApply=从贮藏中重新应用本地更改。
 wndLog.tbtStashSave=贮藏当前的本地更改。
 wndLog.tbtSync=推送当前分支的本地提交并拉取远程程更改。
 wndLog.tbtUnstage=从索引中删除暂存更改。
 wndLog.tbtViewIgnored=如果选中，将显示忽略的文件。
-wndLog.tbtViewStaged=If selected, files with staged \(Index\) changes and without working tree changes will be shown.
+wndLog.tbtViewStaged=如果选中，将显示具有暂存 \(Index\) 更改且未更改工作树的文件。
 wndLog.tbtViewUnchanged=如果选中，将显示未更改的文件。
 wndLog.tbtViewUnchangedAssumed=如果选中，将显示具有 “假设不变” 标志的文件。
-wndLog.tbtViewUnversioned=If selected, not yet version controlled files will be shown.
-wndLog.tbtWindowLayoutSetMain=Switch to the Main perspective.
+wndLog.tbtViewUnversioned=如果选中，则不会显示版本控制的文件。
```

Comment:
I think it is "如果选中，未版本控制的文件将会显示。"

Reply:
@taotieren please also finally squash all your commits to a single commit before updating the pull request, then I can merge as is.

Reply:
Cheers！

Reply:
@taotieren Thanks! It seems that with the merge 24c8da23 something went wrong and `wndLog.tbtViewUnversioned` was set to `如果选中，则不会显示版本控制的文件。` instead of `如果选中，未加入版本控制的文件将会显示。` as suggested by @Kerwin1202. I have now squashed your commits and pushed to temporary branch syntevo:taotieren. You may fetch this branch to your repository, use SmartGit's *Reset* to set your master onto this commit and then send resend the pull request.  If I won't see a pull request within the next couple of days, I will simply cherry-pick the commit (you are still the author, of course).

Reply:
Sure！